manifests/argocd-operator.yaml

```diff
@@ -0,0 +1,3196 @@
+# This yaml is built by this command: kustomize build "https://github.com/argoproj-labs/argocd-operator/deploy?ref=v0.0.14"
```

Comment:
Can we split the CRDS out into the crd phase?

Reply:
I can do that.